packager/certfetcher/certfetcher.go

```diff
@@ -119,15 +119,16 @@ func New(email string, eabKid string, eabHmac string, certSignRequest *x509.Cert
 		}
 	}
 
-	// Theoretically, this should always be set to false as users should have pre-registered for access
-	// to the ACME CA and agreed to the TOS.
-	// TODO(banaag): revisit this when trying the class out with Digicert CA.
-	if !shouldRegister {
```

Comment:
Remove the shouldRegister param; no longer used.

Reply:
Actually, we need to keep it so that we can skip registration during tests. Modified the code flow accordingly.